holdem/env.py

```diff
@@ -146,17 +146,16 @@ def reset(self):
     self._ready_players()
     self._number_of_hands = 1
     [self._smallblind, self._bigblind] = TexasHoldemEnv.BLIND_INCREMENTS[0]
-    if (self.emptyseats < len(self._seats) - 1):
+    if len(self._player_dict) >= 2:
       players = [p for p in self._seats if p.playing_hand]
       self._new_round()
-      self._round = 0
       self._current_player = self._first_to_act(players)
       self._post_smallblind(self._current_player)
       self._current_player = self._next(players, self._current_player)
       self._post_bigblind(self._current_player)
       self._current_player = self._next(players, self._current_player)
       self._tocall = self._bigblind
-      self._round = 0
+      self._round = 0 # 1-preflop, 2-flop, 3-turn, 4-river, 5-showdown
```

Comment:
It might be worth making this an `IntEnum`, too.

Reply:
Dealt with!